### PR TITLE
Profile page UI

### PR DESCRIPTION
**Pull request by Priya-Patel31**
Added profile component and navigation for the left sidebar .

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,14 +1,22 @@
 import React from "react";
-
+import { Routes, Route } from "react-router-dom";
 import "./App.css";
 import Home from "./pages/home/Home";
+import Profile from "./pages/profile/Profile";
+import Feeds from "./pages/feed/Feeds";
```
Comment (t007rushi) [inline src/App.tsx:6]: barrel export suggested

File: src/pages/profile/profile.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+.profile-username {
+  letter-spacing: 1px;
+}
+.profile-bio {
+  margin-top: 2rem;
+  margin-bottom: 1rem;
+}
+.edit-button {
+  padding: 0.4rem 0.8rem;
+  margin: auto !important;
```
Comment (t007rushi) [inline src/pages/profile/profile.css:32]: try to avoid ```!important```

File: src/shared/leftSidebar/LeftSidebar.tsx [hunk 3]
```diff
@@ -6,26 +6,35 @@ import {
...
       <ul className="leftSidebar-list-items list-style-none">
-        <li className="leftSidebar-list-item active font-bold">
-          <AiFillHome className="leftSidebar-list-icon" />
-          <a href="/" className="list-item-label">Home</a>
+        <li className="leftSidebar-list-item font-bold">
+          <NavLink to="/" className="flex-row align-center">
+            <AiFillHome className="leftSidebar-list-icon" />
+            <span className="list-item-label">Home</span>
+          </NavLink>
         </li>
```
Comment (t007rushi) [inline src/shared/leftSidebar/LeftSidebar.tsx:20]: can map it in array of options as to follow ```DRY```

**Review by t007rushi — APPROVED** [review]
There are suggestions else good to merge.

File: src/index.tsx [hunk 4]
```diff
@@ -1,18 +1,21 @@
-import React from 'react';
-import { createRoot } from 'react-dom/client';
-import { Provider } from 'react-redux';
-import { store } from './app/store';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
-import './index.css';
+import React from "react";
```
Comment (tusharkandpal) [inline src/index.tsx:1]: this is no longer needed to import.

**Review by tusharkandpal — APPROVED** [review]
looks good so far. can merge 👍🏻

File: src/pages/profile/profile.css (hunk 2 above)
Comment (Priya-Patel31) [reply src/pages/profile/profile.css:32]: Here it is necessary as some CSS was overriding .

File: src/shared/leftSidebar/LeftSidebar.tsx (hunk 3 above)
Comment (Priya-Patel31) [reply src/shared/leftSidebar/LeftSidebar.tsx:20]: Yes I will do later when I will add functionality.

File: src/index.tsx (hunk 4 above)
Comment (Priya-Patel31) [reply src/index.tsx:1]: I will remove it .

**Pull request merged**